openslides/core/views.py

```diff
@@ -24,6 +25,14 @@
 from .serializers import CustomSlideSerializer, TagSerializer
 
 
+class AngularView(utils_views.View):
```

Comment:
At the moment the file is fetched from a fixed location in the core. Plugins can't override it if not using an extra webserver like Apache2 or Nginx. Hasn't Django a smart way to deliver static files using the static file finders first? Maybe something like https://docs.djangoproject.com/en/1.7/ref/contrib/staticfiles/#finders-module? I can look for it if you don't have enough time.